src/views/Home.vue

```diff
@@ -179,12 +181,16 @@ ion-card > ion-card-header {
 .scroller-content {
   display: flex;
   overflow-x: scroll;
+  overscroll-behavior-x: contain;
+  scroll-snap-type: x mandatory;
 }
 
 .scroller-item{
   flex-shrink: 0;
   position: relative;
+  width: max-content;
```

Comment:
why do you need this? what happens if you don't do this?

Reply:
I was trying a few things and by mistake I forgot to remove this line. It is of no use and I have removed it.